web/cobrands/fixmystreet/assets.js

```diff
@@ -796,22 +800,25 @@ return {
 })();
 
 $(fixmystreet).on('body_overrides:change', function() {
-    var councils_text = $('#js-councils_text').html();
+    var text = $('#js-councils_text').html(),
+        single_body_only = $('#single_body_only').val(),
+        do_not_send = $('#do_not_send').val();
 
-    var single_body_only = $('#single_body_only').val();
     if (single_body_only) {
-        councils_text = councils_text.replace(/<strong>.*<\/strong>/, '<strong>' + single_body_only + '</strong>');
+        text = text.replace(/<strong>.*<\/strong>/, '<strong>' + single_body_only + '</strong>');
+    } else {
+        // Reset back to list of bodies first
+        var bods = fixmystreet.bodies.join('</strong> ' + translation_strings.or + ' <strong>');
+        text = text.replace(/<strong>.*<\/strong>/, '<strong>' + bods + '</strong>');
     }
 
-    var do_not_send = $('#do_not_send').val();
     if (do_not_send) {
         do_not_send = fixmystreet.utils.csv_to_array(do_not_send);
         for (var i=0; i<do_not_send.length; i++) {
-            // XXX Translations
-            councils_text = councils_text.replace(new RegExp('or <strong>' + do_not_send[i] + '</strong>'), '');
-            councils_text = councils_text.replace(new RegExp('<strong>' + do_not_send[i] + '</strong> or '), '');
+            text = text.replace(new RegExp(translation_strings.or + '<strong>' + do_not_send[i] + '</strong>'), '');
+            text = text.replace(new RegExp('<strong>' + do_not_send[i] + '</strong>' + translation_strings.or), '');
         }
     }
 
-    $('#js-councils_text').html(councils_text);
+    $('#js-councils_text').html(text);
```

Comment:
Given some cobrands don't load `assets.js` I didn't want to put asset specific stuff elsewhere, and keep the file self-contained. But yes, it could probably be done better.